### PR TITLE
add readme

### LINKED ISSUE #3: README needed
Hi chaps,

I'd suggest creating a readme to help your users - this would ideally (IMHO) be something like we have on the other repos:

https://github.com/openfaas-incubator/golang-http-template

What do you think?

Cheers,

Alex

### PR DESCRIPTION
**Pull request by burtonr**
Signed-off-by: Burton Rheutan <rheutan7@gmail.com>

Resolves #3 

Adding a README file with an example

### CODE REVIEW

**Comment by burtonr** [comment]
@alexellis I'd appreciate your feedback :smile:

File: README.md [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+namespace Function
+{
+    public class FunctionHandler
+    {
+        public Task<string> Handle(string input)
+        {
+            StringBuilder sb = new StringBuilder();
+            StringWriter sw = new StringWriter(sb);
+
+            using (JsonWriter writer = new JsonTextWriter(sw))
```
Comment (alexellis) [inline README.md]: Possible `var writer`? and `var` for `sb` / `sw`?

**Comment by alexellis** [comment]
I like it a lot, LGTM!

File: README.md [hunk 2]
```diff
@@ -0,0 +1,80 @@
...
+            {
+                writer.Formatting = Formatting.Indented;
+
+                writer.WriteStartObject();
```
Comment (alexellis) [inline README.md]: Would a more common example be something like calling Serialize?

Comment (burtonr) [reply README.md]: Good call! I've updated the sample to a more simple `JsonConvert.SerializeObject`.

**Pull request merged**